src/createStore.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+/**
+ * Implement `createStore` function:
+ *
+ * Function takes 2 arguments and returns an object
```

Comment:
Function takes 2 arguments: `recuder` and `initialState` and returns an object with 3 methods:
 - `getState` returns current state
 - `dispatch` accepts an `action` object. It runs the `reducer` passing current state and the `action`. New state returned by the `reducer` is saved to be returned by `getState`
 - `subscribe` accepts a callback. All the added callbacks are run without arguments each time after the `dispatch` updates the `state`.